app/styles/application-launcher.less

```diff
@@ -0,0 +1,51 @@
+//
+// Application Launcher
+// --------------------------------------------------
+
+.navbar-pf,.navbar-pf-alt {
+  .applauncher-pf {
+    .applauncher-pf-icon {
+      @media (min-width: @screen-sm-min) {
+        font-size: 16px;
+      }
+    }
+  }
+}
+
+.nav.navbar-iconic {
+  .applauncher-pf {
+    .dropdown-menu {
```

Comment:
```less
.nav.navbar-iconic .applauncher-pf .dropdown-menu {
   ...
}
```

Same comment for the styles below. We try to follow the Patternfly guide for markup and CSS.

http://codeguide.patternfly.org/#css-nesting